### DISCUSSION TITLE
Access control for Views [Jul 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2869] [Discord Threads]
> 🙋‍♂️ **rohitstartsup**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Access control for views is supported. You can configure views to be allowed or blocked for a specific role.  
![image0](https://cdn.discordapp.com/attachments/999307820350570556/999308785350889482/Screenshot_2022-07-20_at_7.05.43_PM.png)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999307820350570556)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
